core/modules/simpletest/tests/menu.test

```diff
@@ -49,7 +49,7 @@ class MenuWebTestCase extends BackdropWebTestCase {
     $pass = ($pass && empty($parts));
 
     $this->assertTrue($pass, format_string('Breadcrumb %parts found on @path.', array(
-      '%parts' => implode(' » ', $trail),
+      '%parts' => implode(' ', $trail),
```

Comment:
Also, you can use core's `debug()` function to output any message directly to the results table of a test in the UI, which can be helpful for debugging tests.